tests/test_visualization.py

```diff
@@ -1,18 +1,96 @@
+from typing import List  # NOQA
+from typing import Optional  # NOQA
+
+from optuna.distributions import CategoricalDistribution
 from optuna.distributions import UniformDistribution
+from optuna.structs import StudyDirection
 from optuna.study import create_study
+from optuna.study import Study  # NOQA
 from optuna.trial import Trial  # NOQA
+from optuna.visualization import _generate_contour_subplot
 from optuna.visualization import _get_contour_plot
 from optuna.visualization import _get_intermediate_plot
 from optuna.visualization import _get_optimization_history_plot
 from optuna.visualization import _get_parallel_coordinate_plot
 from optuna.visualization import _get_slice_plot
 
+import pytest
+
+
+def prepare_study_with_trials(no_trials=False, less_than_two=False, with_c_d=True):
```

Comment:
Please add `_` prefix to the name of this private method:
```suggestion
def _prepare_study_with_trials(no_trials=False, less_than_two=False, with_c_d=True):
```